concordium-consensus/src/Concordium/Skov/MonadImplementations.hs

```diff
@@ -48,54 +49,42 @@ import Concordium.Skov.Update
 import Concordium.TimeMonad
 import Concordium.TimerMonad
 
--- |Base monad that provides: IO, logging, global-state context and global-state state.
-type RWSTIO c pv = RWST (Identity (GSContext c pv)) () (Identity (GSState c pv)) LogIO
+-- |Monad that provides: IO, logging, global-state context, global-state state and SkovQueryMonad via SkovQueryMonadT.
```

Comment:
Does this comment still make sense? What is global-state context and state referring to?

Reply:
I have update the comment